topics/go/algorithms/data/hash/hash_test.go

```diff
@@ -0,0 +1,63 @@
+package hash
+
+import (
+	"testing"
+)
+
+func TestFunction(t *testing.T) {
```

Comment:
The tests need to follow the semantics/mechanics found here for consistency.

https://github.com/ardanlabs/gotraining/blob/master/topics/go/algorithms/strings/reverse/reverse_test.go

Reply:
OK

Reply:
Is this the right link? Not sure how to do table driven tests here (in a simple manner).

Reply:
We don't need to do a table test, jsut the rest of the formating.

Reply:
Let's fix this when coding together